people/barnabyjack.json

```diff
@@ -5,7 +5,7 @@
 	"birth" : "1977",
 	"death" : "2013",
 	"issue" : "4",
-	"mainimage" : "/images/y3t1_Art-Barnaby_Jack.png",
```

Comment:
First of, this is all great, thank you for doing this. One small nit pick is that we want y3ti's art to keep it's designation for attributions sake.

Reply:
Cool, didn't realize it, fixed that with: https://github.com/restincode/restincode/pull/129/commits/e5dada05bf14b4f1c6e241174e6b4440b9d18b62